docs/dev/metas_api/blockmeta.rst

```diff
@@ -0,0 +1,7 @@
+BlockMeta
+---------
+
+.. module:: malcolm.metas.blockmeta
```

Comment:
Can you make it so that the import path is as we end up using it. If you just make all the metas `.. module:: malcolm.metas` then this should work